passes/source2il.nim

```diff
@@ -24,10 +24,13 @@ type
     tkInt
     tkFloat
 
-  Context = object
+  Context* = object
```

Comment:
```suggestion
  ModuleCtx* = object
```

You don't have to make the change, just thought it was slightly more descriptive as to what it is.